core/merginapi.cpp

```diff
@@ -29,8 +29,10 @@
 #include <geodiff.h>
 
 const QString MerginApi::sMetadataFile = QStringLiteral( "/.mergin/mergin.json" );
+const QString MerginApi::sMerginConfigFile = QStringLiteral( "/.mergin-config.json" );
```

Comment:
Maybe we should also place it to `.mergin` subfolder?

Reply:
I was thinking about it, but the config file cannot be excluded from the sync.